app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -77,13 +71,14 @@ const Notes = () => {
         </div>
       )}
       <NewNotePane
-        fetchNotes={fetchNotes}
+        notes={notes}
+        setNotes={setNotes}
         setShowPane={setShowNewNotePane}
         showPane={showNewNotePane}
       />
       {showDeleteAlert && (
         <DeleteAlert
-          refetch={fetchNotes}
+          refetch={undefined}
```

Comment:
Why have you passed an undefined value for this prop? If it's not being used please remove it.